[IndexFilters] Loading spinner moved to be a suffix within the search box. (#10808)
Shopify/polaris: .changeset/sixty-fishes-grin.md

---
'@shopify/polaris': patch
---

> [IndexFilters] Loading spinner moved to be a suffix within the search box.

Reviewer: ```suggestion
Moved the `Filters` loading spinner to the suffix of the query filter
```
